DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Delegates/EditDelegateControllerTests.cs

```diff
@@ -113,17 +153,70 @@ public void Index_post_returns_view_with_model_error_with_invalid_prn()
                     result,
                     "Invalid professional registration number format - Only alphanumeric characters (a-z, A-Z and 0-9) and hyphens (-) allowed"
                 );
+                A.CallTo(() => userService.GetDelegateById(A<int>._)).MustNotHaveHappened();
             }
         }
 
         [Test]
-        public void Index_post_calls_userService_and_redirects_with_no_validation_errors()
+        public void Index_post_calls_userServices_and_redirects_with_no_validation_errors()
         {
             // Given
+            const string centreSpecificEmail = "centre@email.com";
+            var delegateUser = UserTestHelper.GetDefaultDelegate(

```

Comment:
`var delegateUser` is a `Delegate` rather than a `DelegateUser`, so should be renamed